universities/university-of-illinois-urbana-champaign.md

```diff
@@ -0,0 +1,17 @@
+
+# University of Illinois Urbana-Champaign
```

Comment:
I feel like this entry has to mention Dan Katz.

Reply:
See new changes below